librz/core/casm.c

```diff
@@ -797,7 +795,8 @@ static RzList *rz_core_asm_back_disassemble(RzCore *core, ut64 addr, int len, ut
 		rz_asm_set_pc(core->rasm, current_instr_addr);
 		current_instr_len = next_buf_pos - current_buf_pos;
 		current_instr_len = rz_asm_disassemble(core->rasm, &op, buf + current_buf_pos, current_instr_len);
-		IFDBG {
+#if 0
```

Comment:
Maybe we need `RZ_LOG_TRACE()` for such code pieces.

Reply:
that piece of code for me can be deleted.

Reply:
Fine for me, just remove it then please